### PR TITLE
Doc Aurora CDC

### LINKED ISSUE #1932: Change required: Aurora MySQL/PostgreSQL sources
### Related code PR

Please add Aurora MySQL/PostgreSQL sources docs

### Which part(s) of the docs might be affected or should be updated? And how?

https://docs.risingwave.com/docs/current/ingest-from-postgres-cdc/#set-up-postgresql
https://docs.risingwave.com/docs/current/ingest-from-mysql-cdc/

Add Aurora MySQL/PostgreSQL sources pages. You can reuse the above page for the Aurora PostgreSQL/MySQL part. Because most steps after setup the AWS Aurora are the same as the above pages.

Refer to our test steps https://www.notion.so/risingwave-labs/Aurora-CDC-Test-e09aa9621df243c5a9e36e97a2ec492a

### Reference

_No response_

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  - Add AWS Aurora to set-up for MySQL and PG CDC guides. 
 
- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1932

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Comment by emile-00** [comment]
@cyliu0 I only adjusted the tab title under "Set up" to include Aurora since the instructions seem the same as AWS RDS. If Aurora should have a separate tab, please let me know. Thanks!

File: docs/guides/ingest-from-mysql-cdc.md [hunk 1]
```diff
@@ -96,9 +96,9 @@ SHOW VARIABLES LIKE 'log_bin';
 See [Setting up MySQL](https://debezium.io/documentation/reference/stable/connectors/mysql.html#setting-up-mysql) for more details.
 
 </TabItem>
-<TabItem value="AWS RDS MySQL" label="AWS RDS">
+<TabItem value="AWS/Aurora MySQL" label="AWS RDS and Aurora">
```
Comment (cyliu0) [inline docs/guides/ingest-from-mysql-cdc.md]: In fact, Aurora and MySQL are AWS RDS's engine options. So you might need to change `AWS RDS and Aurora` to `AWS RDS MySQL and Aurora (MySQL-Compatible)`

File: docs/guides/ingest-from-mysql-cdc.md [hunk 2]
```diff
@@ -96,9 +96,9 @@ SHOW VARIABLES LIKE 'log_bin';
...
 
-If your MySQL is hosted on AWS RDS, the configuration process is different. We will use a standard class MySQL instance without Multi-AZ deployment for illustration.
+If your MySQL is hosted on AWS RDS or Aurora, the configuration process is different. We will use a standard class AWS RDS MySQL instance without Multi-AZ deployment for illustration, but the process will be similar for Aurora.
```
Comment (cyliu0) [inline docs/guides/ingest-from-mysql-cdc.md]: ditto

File: docs/guides/ingest-from-postgres-cdc.md [hunk 3]
```diff
@@ -105,9 +105,9 @@ import TabItem from '@theme/TabItem';
     ```
 
 </TabItem>
-<TabItem value="AWS_rds_pg" label="AWS RDS">
+<TabItem value="AWS_rds_pg" label="AWS RDS and Aurora">
```
Comment (cyliu0) [inline docs/guides/ingest-from-postgres-cdc.md]: ditto

**Review by cyliu0 — APPROVED** [review]

**Review by ShanlanLi — APPROVED** [review]
LGTM

**Pull request merged**